src/main/java/com/app/medicalwebapp/services/FileService.java

```diff
@@ -28,9 +30,27 @@ public FileService(FileSaverStrategyResolver saverStrategyResolver, FileExtracto
      * Сохранение файла.
      */
     public FileObject saveFile(String originalName, byte[] fileContent, Long ownerId, String UID) throws Exception {
-        FileSaverStrategy fileSaver = saverStrategyResolver.getFileSaver(originalName); // Выбрать способ сохранения файла, зависит от его расширения.
-        FileObjectFormat format = FileFormatResolver.resolveFormat(originalName);
-        return fileSaver.save(ownerId, originalName, format, fileContent, UID);
+        // Проверка файлов на идентичные загруженные файлы на сервере
+        var savedFiles = fileObjectRepository.findByOwnerAndDeleted(ownerId, false);
+        var element = savedFiles.stream()
+                    .filter(x ->
+                    {
+                        try{
+                            return (Arrays.equals(fileContent, previewFile(x)));
+                        } catch(Exception e) {

```

Comment:
```suggestion
                        } catch (Exception e) {
```
Вообще, Exception ловить нельзя, а то много чего наловится.